sdk/tvOS/OoyalaTVSkinSDK/OOOoyalaTVButton.m

```diff
@@ -0,0 +1,39 @@
+//
+//  OOOoyalaTVButton.m
+//  OoyalaTVSkinSDK
+//
+//  Created by Yi Gu on 7/19/16.
+//  Copyright © 2016 ooyala. All rights reserved.
+//
+
+#import "OOOoyalaTVButton.h"
+
+@implementation OOOoyalaTVButton
+
+- (id)initWithFrame:(CGRect)frame {
+  self = [super initWithFrame:frame];
+  
+  return self;
+}
+
+- (void)setupPlayPause: (BOOL)isPlay {
+  NSString *fontString = isPlay ? @"g": @"v";
```

Comment:
Can you instead, configure everything on initWithFrame, and only change the font text in this method?

 At that point, please rename the method to "changePlayingState", or something similar.

The method should represent a stateful action (changePlayingState), and the TVButton decides what to do based on the input (showing the Play or Pause button)